src/interfaces/node.h

```diff
@@ -204,7 +204,7 @@ class Node
     virtual std::unique_ptr<Wallet> loadWallet(const std::string& name, std::string& error, std::vector<std::string>& warnings) = 0;
 
     //! Create a wallet from file
-    virtual WalletCreationStatus createWallet(const SecureString& passphrase, uint64_t wallet_creation_flags, const std::string& name, std::string& error, std::vector<std::string>& warnings, std::unique_ptr<Wallet>& result) = 0;
+    virtual std::unique_ptr<Wallet> createWallet(const SecureString& passphrase, uint64_t wallet_creation_flags, const std::string& name, std::string& error, std::vector<std::string>& warnings, WalletCreationStatus& status) = 0;
```

Comment:
nit: considering a new signature of the refactored function could a more appropriate semantically name be chosen?
`newWallet` or smth similar?

Reply:
re: https://github.com/bitcoin/bitcoin/pull/18278#discussion_r389316112

> nit: considering a new signature of the refactored function could a more appropriate semantically name be chosen?
> `newWallet` or smth similar?

Unclear why that name is more semantically appropriate, but not knowing this, it seems reasonable for the interface wrapper method name to have the same name as the function it is wrapping